### DISCUSSION TITLE
Mouse brain representation using SIMNIBS

### DISCUSSION THREAD

**Discussion by anita0802** [simnibs/simnibs#188] [7.Creating head models] [answered]
Hi,

I have a problem with SIMNIBS 4.0.1. I need to use it with a mouse brain mesh. I have generated a .msh using Gmsh program, saving it as "version 2 binary" with only "Save all parameters" marked because if I check the "Save parametric coordinates" it fails when I try to load into SimNIBS.

So, when I load only that .msh model to SimNIBS, it does not represent anything. If I copy that .msh into the folder of m2m_ernie it does not do anything.

Could you please help me to load my mouse brain .msh model so that I can use your program?

**Comment by TorgeW** [comment]
Could you please supply the mouse head model so we can check it out?

**Reply by anita0802** [reply]
Yes, here you have the .msh of a mouse brain.
[ImageToStl_wholebrain.zip](https://github.com/simnibs/simnibs/files/12537790/ImageToStl_wholebrain.zip)

**Reply by oulap** [reply]
It seems the mesh you sent includes only the gray matter surface. simnibs requires a full volumetric mesh, i.e., also the tetrahedral elements and not only the triangles. Additionally, you might want to add other tissues, such as the csf, skull and scalp, depending on what type of a simulation you're aiming at.

If you have a segmentation of the anatomy in voxel space, you can create a simnibs compatible mesh from that using the instructions here:

https://simnibs.github.io/simnibs/build/html/tutorial/advanced/custom_meshing.html

**Reply by anita0802** [reply]
And there is any web where I can find the complete mouse brain mesh for using with SimNIBS? Or where I can find the different layers that I need to complete the brain?

Thank you very much in advance.

**Reply by oulap** [reply, marked as answer]
There is a publicly available mouse mesh here:

https://zenodo.org/record/4009465#.X2OFcGhKhPY

if that's suitable for your needs. If you have an MR scan of the mouse, which has been segmented to the different tissues classes, you can generate a mesh from that (using the instructions behind the link I sent previously). If not, you can start with the mouse mesh behind the link above.

**Reply by anita0802** [reply]
Thank you very much for the information. I would try to do the test with that mesh.
